### ISSUE TITLE
Dates in relations

### ISSUE THREAD

**Issue by deternitydx** [snac-cooperative/snac#48]
In gitlab by @deternitydx on Aug 22, 2017, 14:57

IC Relations need to have dates added to them.  They also need to be evidenced in the edit user interface.

**Comment by dpitti** [comment]
Yes, and we also need to discuss the possibility of qualifying by place, as well. Not to be confused with describing a person in association with a place. Many relations may be place-specific. The issue is whether place would be related to our geographic place table.
